drivers/gpio/gpio_lpc11u6x.c

```diff
@@ -141,23 +139,27 @@ static int gpio_lpc11u6x_pin_configure(const struct device *port,
 	if (flags & GPIO_SINGLE_ENDED) {
 		/* Open source mode is not supported. */
 		if (flags & GPIO_LINE_OPEN_DRAIN)
-			func |= IOCON_OPENDRAIN_EN;
+			func |= IOCON_PIO_OD(1);
 		else
 			return -ENOTSUP;
 	}
 
 	if (flags & GPIO_PULL_UP) {
-		func |= IOCON_MODE_PULLUP;
+		func |= IOCON_PIO_MODE(0x1);
 	} else if (flags & GPIO_PULL_DOWN) {
-		func |= IOCON_MODE_PULLDOWN;
+		func |= IOCON_PIO_MODE(0x2);
```

Comment:
The value for the pull-down resistor is 0x1

Reply:
Fixed in latest push